src/common/components/header/index.js

```diff
@@ -2,74 +2,105 @@ import React, { Component, PropTypes } from 'react';
-            <a href="#navigation" className={ style['p-navigation__toggle--open'] } title="menu">Menu</a>
-            <a href="#navigation-closed" className={ style['p-navigation__toggle--close'] } title="close menu">Close menu</a>
+        <div className={ style['p-navigation__banner'] }>
+          <div className={ style['p-navigation__logo'] }>
+            <a className={ style['p-navigation__link'] } href="https://snapcraft.io">
+              <img className={ style['p-navigation__image'] } src={ brandmark } alt="Snapcraft" />
+            </a>
           </div>
-          <nav className={ style['p-navigation__nav'] } role="menubar">
-            <ul className={ style['p-navigation__links']} role="menu">
-              <li className={ style['p-navigation__link'] } role="menuitem">
-                <a href="https://snapcraft.io/store/">Store</a>
-              </li>
-              <li className={ style['p-navigation__link'] } role="menuitem">
-                <a className={ style['is-selected'] } href="/">Build</a>
-              </li>
-              <li className={ style['p-navigation__link'] } role="menuitem">
-                <a className={ style['p-link--external'] } href="https://dashboard.snapcraft.io/snaps">My snaps</a>
-              </li>
-              <li className={ style['p-navigation__link'] } role="menuitem">
-                <a href="https://docs.snapcraft.io">Docs</a>
-              </li>
-              <li className={ style['p-navigation__link'] } role="menuitem">
-                <a className={ style['p-link--external'] } href="https://forum.snapcraft.io/categories">Forum</a>
-              </li>
-            </ul>
-            { authenticated
-              ? (
-                <ul className={ style['p-navigation__links--right']} role="menu">
-                  <li className={ style['p-navigation__item'] } role="menuitem">
-                    <a>Hi, {user.name || user.login}</a>
-                  </li>
-                  <li className={ style['p-navigation__link'] } role="menuitem">
-                    <a href="/auth/logout" onClick={ this.onLogoutClick.bind(this)}>
-                      Sign out
-                    </a>
-                  </li>
-                </ul>
-              ) : (
-                <ul className={ style['p-navigation__links--right']} role="menu">
-                  <li className={ style['p-navigation__link'] } role="menuitem">
-                    <a href="/auth/authenticate">Sign in with GitHub</a>
-                  </li>
-                </ul>
-              )
-            }
-          </nav>
+
+          <a href="#navigation" className={ style['p-navigation__toggle--open'] } title="menu">Menu</a>
+          <a href="#navigation-closed" className={ style['p-navigation__toggle--close'] } title="close menu">Close menu</a>
         </div>
+        <nav className={ style['p-navigation__nav'] } role="menubar">
+          <ul className={ style['p-navigation__links']} role="menu">
```

Comment:
I see that some old inconsistency got copied around and spreads too much. Could we fix the missing space between closing brackets `]}`? It's around in at least several classNames now...